src/web-ui/src/public/Welcome.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+  <Layout :showNav="false" :showFooter="false" backgroundColor="var(--aws-squid-ink)">
+    <div class="container mb-2 text-left">
+      <header class="px-3 px-md-5">
+        <h1 class="heading my-5 text-center">Welcome to the Retail Demo Store!</h1>
+
+        <p>
+          The Retail Demo Store is a demo platform to demonstrate how AWS infrastructure and services can be used to
+          build personalized customer experiences across multiple channels of engagement.
+        </p>
+
+        <p>
+          The Retail Demo Store mimics an online store with products, users and user-interactions to showcase the
+          personalized product recommendations a shopper would receive based on historical and real time click-through
+          behavior.
+        </p>
+
+        <div class="mt-2 d-flex flex-column flex-lg-row align-items-lg-center">
+          <div class="mb-2 mb-lg-0">AWS Services Included:</div>
+
+          <ul
+            class="service-list mb-0 pl-0 flex-grow-1 d-flex flex-column flex-lg-row justify-content-lg-around font-weight-bold"
+          >
+            <li v-for="service in services" :key="service.text" class="mb-2 mb-lg-0">
+              <ServiceBadge :text="service.text" :src="service.src"></ServiceBadge>
+            </li>
+          </ul>
+        </div>
+
+        <!-- make both links to go /auth until we have dedicated sign in and create account pages -->
+        <div class="auth-cta my-5 d-flex flex-column flex-sm-row justify-content-end align-items-center">
+          <div class="login-cta d-flex align-items-center">
+            *Have an account?<router-link to="/auth" class="sign-in ml-3 btn btn-link">Sign in</router-link>
+          </div>
+          <router-link to="/auth" class="create-account mt-3 mt-sm-0 ml-sm-3 btn btn-primary">Create an account</router-link>
+        </div>
+      </header>
+
+      <section class="info py-3 px-3 py-md-4 px-md-5">
+        <p>
+          *To keep the click-through data fresh, this demo resets every day at 11 pm EST. Accounts created more than 24
+          hours prior would be deleted as well. If your account has been deleted, go ahead and create a new one.
```

Comment:
I think we'll want to have this be more dynamic. Most deployments of this project will not be using the reset functionality at all. For example, if I deploy directly from GH. Also, and this is an open question for @alexchirayath, will the reset functionality be unique to the always-on mlsld deployment or something that can be optionally setup for any deployment? Finally, 11pm ET seems kind of early. Should that be configurable and then be dynamically displayed here?

Reply:
Thats a fair point. This probably should not be incorporated into the public demo.
The periodic reset mechanism is built into the deployment architecture (with CodeBuild and Step functions) which is outside of the retail demo store scope.

Reply:
Removed the section for now.